teacher.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'dbc_person'
+
+class Teacher < DbcPerson
+  attr_reader :salary, :target_raise
+
+	def initialize(options={})
+		super
+		@phase = 3
+	end
+
+	def set_phase(num)
+    @phase = num
+    "Cool, I've always wanted to teach phase #{num}!"
+  end
+
+  def teach_stuff
+  	self.class::RESPONSE
+  	# "Listen, class, this is how everything works.#{, fo SHO! *drops #{flat-out insane} knowledge bomb* ... You're welcome. #{*saunters away*}"
+  end
+
```

Comment:
Check the indentation in this file.

Make sure to set sublime (or whatever text editor you're using) to have 2-space tabs, since that's what we use in the files we give you, and we like consistency (also, sometimes if you have tabs in some places and spaces in others, it can look right to you, but then look wrong on github).

http://stackoverflow.com/questions/9474090/how-do-i-force-sublime-text-2-to-indent-two-spaces-per-tab